react/react2/week1/github-searcher-app/src/DisplaySearchResult.js

```diff
@@ -0,0 +1,27 @@
+import { useContext } from "react";
+import { FetchedContext } from "./MainComponent";
+
+export const DisplaySearchResult = () => {
+  const getData = useContext(FetchedContext);
+  const newUser = getData.queryResult.map((item) => <div>{item}</div>);
```

Comment:
You are getting a key warning because you are not passing a key prop to the element of this map. You can read more of that here https://reactjs.org/docs/lists-and-keys.html